src/ot_orchestration/dags/finngen_ingesion.py

```diff
@@ -15,18 +15,10 @@
     install_dependencies,
     submit_step,
 )
+from ot_orchestration.utils.utils import read_yaml_config
```

Comment:
```suggestion
from ot_orchestration.utils import read_yaml_config
```

You export `read_yaml_config` in the __init__file.

Reply:
you are right, I am not too consistent about my `__all__` flags :p. This was IDE that pointed be out to the wrong subpackage.